test/e2e/common_test.go

```diff
@@ -182,8 +179,6 @@ var _ = SynchronizedBeforeSuite(func() []byte {
 
 	return []byte(globalTmpDir)
 }, func(data []byte) {
-	cwd, _ := os.Getwd()
-	INTEGRATION_ROOT = filepath.Join(cwd, "../../")
```

Comment:
This is breaking parallel runs, if you touch these things you should be familiar with how the parallel ginkgo nodes work.
The first function is only run on node 1 while the second one is run on all nodes, so yes this duplication is indeed required here.


One example https://github.com/containers/podman/commit/5eb99a0ac88a6d65d11d307c87043f42b0df07be